src/css/_global.scss

```diff
@@ -21,8 +22,7 @@ code,
 kbd,
 samp,
 tt {
-  // stylelint-disable-next-line font-family-no-duplicate-names
-  font-family: monospace, monospace;
+  font-family: SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace;
```

Comment:
Yeah, it feels better than the default `monospace` in case monospace is needed outside of the code view.

I could put it in a custom property to avoid the repetition if you have strong feels about it.

Reply:
Nah, all good, I figured that these selectors are for normalization.